webview/src/experiments/components/table/styles.module.scss

```diff
@@ -649,6 +629,10 @@ $bullet-size: calc(var(--design-unit) * 4px);
       outline-style: dashed;
       outline-color: $header-dnd-outline;
       outline-offset: -4px;
+
+      .iconMenu ul[role='menu'] {
```

Comment:
Originally, a drag target with an icon looked like this:

<img width="267" alt="image" src="https://user-images.githubusercontent.com/43496356/216124369-b3576993-fe46-4cff-a500-20fc6a520705.png">


Was this intentional since hovering over this button actually won't cause the target to drop? I thought it looked a little buggy so I made the button color transparent when the cell is a drop target, but I can easily revert the change if preferred :)

Reply:
thanks @julieg18, I think it looks better with the fix. I think it should also allow dropping there tbh (no need to fix it if it's not a single obvious line)

Reply:
> thanks @julieg18, I think it looks better with the fix. I think it should also allow dropping there tbh (no need to fix it if it's not a single obvious line)

Might be doable with `pointer-events: none;` and a little JS 🤔, will try to look into it if I have some time one of these days :)